Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
_Would this work if you set VS to use UNIX line endings?_
No, that doesn't seem to work, because every time you fetch the project the setting for the current file is lost. It's git that decides what line endings to use when it writes out the files. In general, it's fine to use the default line ends for the operating system, but plottable's gruntfile assumes the line ends are \n, regardless of the operating system.
_I would prefer not having Windows endings as well in our files if we can help it._
So -  a better solution than what I have proposed here may to to introduce a .gitattributes file. This gives git some project specific information. With .gitattributes, you can force the line ends to use for various file types  - see [here](https://help.github.com/articles/dealing-with-line-endings/).
Is this a better approach, and if so, how would you prefer to move forward with this?